case-lib/lib.sh

```diff
@@ -40,10 +40,26 @@ if [ ! "$SOFCARD" ]; then
 		awk '/sof-[a-z]/ && $1 ~ /^[0-9]+$/ { $1=$1; print $1; exit 0;}')
 fi
 
-func_lib_setup_kernel_checkpoint()
+setup_kernel_check_point()
 {
-    # shellcheck disable=SC2034 # external script will use it
-    KERNEL_CHECKPOINT=$(date +%s)
+    local check_point
+    check_point="$1"
```

Comment:
This can be a single line because `"$1"` is not a command and has no exit status so SC2155 does NOT apply